app/views/liquids/_reddit.html.erb

```diff
@@ -0,0 +1,33 @@
+<!-- app/views/liquids/_reddit.html.erb -->
```

Comment:
We generally don't have comments about the file's path, and we could have it but that should probably be in a different PR. Also since this is an HTML comment let's remove this one at least. :)